bazel/utils/run_clang_format.template

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
I recommend for templates to have the correct extension, so that editors still pick up the language correctly without manual work, eg, rename to `run_clang_format.template.sh`?